symfony/framework-bundle/3.3/post-install.txt

```diff
@@ -11,3 +11,7 @@
        Run <comment>composer require server</> for a better web server.
 
   * <fg=blue>Read</> the documentation at <comment>https://symfony.com/doc</>
+
+  * <fg=blue>Thanks</> for using Symfony!
+    Spread some love by starring the GitHub repositories you use!
+    Run <comment>composer req --dev thanks && composer thanks</>
```

Comment:
Should be run once *all* deps are installed, not just after the initialization. Not sure if all this is really needed or even desirable.